packages/hackathon/src/scenes/ProjectsScene/ProjectsScene.tsx

```diff
@@ -63,6 +67,23 @@ export const ProjectsScene: FC<ProjectSceneProps> = () => {
     if (hackathon) dispatch(lockProjects(false, hackathon._id))
   }
 
+  useEffect(() => {
```

Comment:
Not sure if useEffect is needed here. Unless we are triggering project scene re-render periodically, this is not needed.

Reply:
I may not understand `useSelector()` well enough yet, but it seems `hackathon` and `hacker` may not be established when this page is first opened. Is that correct?

Reply:
Redux state change will automatically kick off a render. 
`When an action is dispatched, useSelector() will do a reference comparison of the previous selector result value and the current result value. If they are different, the component will be forced to re-render. If they are the same, the component will not re-render.`
https://react-redux.js.org/api/hooks
There's more nuances when dealing with state comparision, but that shouldn't matter here, you can test it to double check.

Reply:
Also afaict, project scene is not rendered or can be navigated to until hacker/hackathon is all loaded in the redux state in the first place. Hence the long lag between app opening and project tab appearing.
(Something to fix later)

Reply:
Good stuff. Thanks for the cleaner version. I put it into this PR.